pages/kids/stories/[story].tsx

```diff
@@ -36,13 +36,14 @@ const StoriesContainer = ({ story, phrases }: StoriesProps): ReactNode => {
 
   const title_current = story.title[currentLanguage] || '';
   const title_other = story.title[otherLanguage] || '';
+  const image_current_story = story.slug;
 
   return (
     <div className="bg-gradient-to-r from-[#fdf6d2] to-[#99bde4] -mb-8 -mt-2 md:p-12 w-full">
       <SEO
-        title={t(`seo.kids_page_storiesTitle.${getCountryVariant()}`)}
+        title={`${title_current + ' - '}` + t(`seo.kids_page_storiesTitle.${getCountryVariant()}`)}
         description={t(`seo.kids_page_storiesDescription.${getCountryVariant()}`)}
-        image="https://www.movapp.cz/kids/hrad.png"
+        image={`https://www.movapp.cz/kids/' + ${image_current_story} + '.jpg'`}
```

Comment:
I see one apostrophe is missing here, please correct that @daria31v @jejdacz 

![image](https://github.com/cesko-digital/movapp/assets/59498/8717d6f5-b607-4f1f-88ee-20e02bcec62b)